game/hud/src/widgets/Crafting/components/Input.tsx

```diff
@@ -135,11 +123,8 @@ class Input extends React.Component<InputProps, InputState> {
 
   private onKeyUp = (e: React.KeyboardEvent<HTMLInputElement>) => {
     if (this.state.changed) {
-      if (e.keyCode >= 32) {
-        this.delayedOnChange(500);
-      }
-      if (e.keyCode === 13) {
-        this.delayedOnChange(0);
+      if (e.keyCode >= 32 || e.keyCode === 13) {
```

Comment:
Can we check these against our jsKeyCodes enum in @csegames/camelot-unchained ? Just makes it easier to understand :p